src/term/term_to_net.rs

```diff
@@ -260,7 +260,7 @@ impl<'a> EncodeTermState<'a> {
       }
       // core: #val
       Term::Num { val } => {
-        // debug_assert!(*val <= LABEL_MASK); // Uneeded?
+        // debug_assert!(*val <= LABEL_MASK); // Unneeded?
```

Comment:
Are we checking somewhere that the number fits in an hvmc node?

Also I think we can remove this line

Reply:
Should we add a check now? i don't think we are